### ISSUE TITLE
Doc string wrongly says dropout is used.

### ISSUE THREAD

**Issue by johnarevalo** [tensorflow/models#5814]
https://github.com/tensorflow/models/blob/23376e62928486712f6a5cff260c93d0b795bdc0/samples/core/get_started/custom_estimator.py#L30

This line mentions a dropout is used. Still, the code doesn't apply dropout anywhere.  is maybe `learning_rate`?

top-level directory: N/A
OS Platform and Distribution: N/A
TensorFlow installed from: N/A
TensorFlow version: N/A
Bazel version: N/A
CUDA/cuDNN version: N/A
GPU model and memory: N/A
Exact command to reproduce: N/A

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
